clippy_lints/src/strings.rs

```diff
@@ -169,3 +169,61 @@ impl<'a, 'tcx> LateLintPass<'a, 'tcx> for StringLitAsBytes {
         }
     }
 }
+
+/// **What it does:** Checks for usage of `.repeat(1)` since its equivalent to clone()
+///
+/// **Why is this bad?** `.repeat(1)` is not very readable.
```

Comment:
Here it might also be better to talk about the hidden intention behind the `repeat(1)` call.

"Isn't very readable" is a very vague argument for a lint.